sw/device/lib/BUILD

```diff
@@ -0,0 +1,106 @@
+# Copyright lowRISC contributors.
```

Comment:
I'm not a huge fan of copying the meson.build files over to bazel BUILD files, at least insofar as leaving meson's machinery around as comments.